Dockerfile

```diff
@@ -0,0 +1,23 @@
+# Use the official Node.js:lst runtime as a base image
+FROM node:lts
```

Comment:
Is it worth it to use a more specific tag? I'm assuming this will eventually get bumped to another version of node, though not anytime soon I'm sure